pkg/utils/utils.go

```diff
@@ -189,3 +197,143 @@ func ConfigureServiceability(pts *corev1.PodTemplateSpec, la *openlibertyv1beta1
 		}
 	}
 }
+
+func getValue(v interface{}) string {
+	switch v.(type) {
+	case string:
+		return v.(string)
+	case bool:
+		return strconv.FormatBool(v.(bool))
+	default:
+		return ""
+	}
+}
+
+func getSsoEnv(loginID string, envSuffix string, value interface{}) *corev1.EnvVar {
```

Comment:
To be consistent with other parts of the code, what do you think about `getSSOEnv?

Reply:
The naming guidelines suggested capitalizing only the first letter of abbreviations that are in the middle. It does look weird. Changed name to `createEnvVarSSO` :)